packages/dx-chart-core/src/plugins/series/computeds.ts

```diff
@@ -97,12 +97,12 @@ getBarPointTransformer.isBroad = true;
 getPiePointTransformer.getPointColor = (palette, index) => palette[index % palette.length];
 
 export const findSeriesByName = (
-  name: string, series: SeriesList,
-) => series.find(seriesItem => seriesItem.symbolName === name);
+  name: symbol, series: SeriesList,
+): Series => series.find(seriesItem => seriesItem.symbolName === name) as Series;
 
 export const dBar = ({
   x, y, y1, width,
-}: TransformedPoint & { width: number }) => ({
+}: { x: number, y: number, y1: number, width: number }) => ({
```

Comment:
Why is it changed?

Reply:
Because there is ts error: missing the extra properties from type 'TransformedPoint': argument, value, index, color.

Reply:
Argument is of `<Connected>BarPoint` type. `TransformedPoint` is the most base type with a required set of properties.